recipes/kissfft/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class KissfftConan(ConanFile):
+    name = "kissfft"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mborgerding/kissfft"
+    description = "a Fast Fourier Transform (FFT) library that tries to Keep it Simple, Stupid"
+    topics = ("conan",)
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "datatype": ["float", "double", "int16_t", "int32_t", "SIMD"],
```

Comment:
```suggestion
               "datatype": ["float", "double", "int16_t", "int32_t", "simd"],
```
CMAKe wants lower case: 

```
CMake Error at source_subfolder/CMakeLists.txt:64 (message):
  Incorrect value of KISSFFT_DATATYPE! It can be one of: double, float,
  int16_t, int32_t, simd
```